### DISCUSSION TITLE
Generate Default Engines Button

### DISCUSSION THREAD

**Discussion by dmue-vbg** [NVIDIA/Stable-Diffusion-WebUI-TensorRT#277] [Q&A]
allow me to ask. where can i find this "Generate Default Engines" button mentioned in the install procedure ?
do i need the latest version of Automatic1111 ?

**Comment by piotrsiwek** [comment]
TensorRT tab, between Train and Settings tabs.

**Reply by dmue-vbg** [reply]
alright, this tab was missing due to invalid installation. thanks a lot!
